package/rmt-server.spec

```diff
@@ -50,6 +50,7 @@ Source14:       nginx-https.conf
 Source15:       auth-handler.conf
 Source16:       auth-location.conf
 Source17:       rmt-cli_bash-completion.sh
+Source18:       rmt-server.reg
```

Comment:
In the hindsight, adding `SourceN` items was perhaps not the best idea. It is easier to just throw those files into the tarball and install from there, rather than maintaining the list of sources.

Thoughts on moving those into the tarball? (I can do that later in a separate PR for some of those sources)

Reply:
I agree, let's do this in a separate PR.